system/settings/win32.xml

```diff
@@ -29,13 +20,6 @@
     </category>
   </section>
   <section id="system">
-    <category id="videoscreen">
-      <group id="1">
-        <setting id="videoscreen.fakefullscreen">
```

Comment:
it should go away from here and all other systems which do not use it: https://github.com/xbmc/xbmc/blob/af7ea36d48d79c02f07c59f4a48340149ed57446/system/settings/settings.xml#L2462

why should Linux bother with a windows feature?